backend/lib/server_code/api/roomRoutes.js

```diff
@@ -38,18 +38,20 @@ roomRouter.post('/create', async (req, res) => {
   }
 });
 
-roomRouter.get('/join', (req, res) => {
-  const { userID, roomID, playerName } = req.query;
-  const playerToAdd = { id: userID, name: playerName };
+roomRouter.get('/register', (req, res) => {
+  console.log(req.query);
+  const { playerID, roomID, playerName } = req.query;
+  const playerToAdd = { id: playerID, name: playerName };
+  console.log(playerToAdd);
   console.log(`roomID:${roomID}`);
   if (Number(roomID) === 1 && roomCreated) {
-    roomPlayers.push(playerToAdd);
+    roomPlayersList.push(playerToAdd);
     const [roomSocket, roomSocketPort] = roomsSockets[roomID];
-    roomSocket.emit('PlayerJoinedRoom', {
-      message: 'A new user has joined the room',
-      roomPlayers,
+    roomSocket.emit('PlayerRegisteredRoom', {
+      message: 'A new player has registered to the room',
+      roomPlayersList,
     });
-    res.send({ joinDate: res.date, roomPlayers, roomSocketPort });
+    res.send({ registerDate: res.date, roomPlayersList, roomSocketPort });
```

Comment:
i don't really think we need the date

Reply:
also, no need to send him the roomPlayersList as we have a socket with that info

Reply:
yes we do, socket is just for updating, the sending of playerlist is for init.
but you are right about the date so i delete it.

Reply:
fe277e0